docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       db:
         condition: service_healthy
   db:
-    image: neo4j:3.4-enterprise
+    image: neo4j:3.5.26
```

Comment:
Were you able to test a query by any chance? I'm just worried that the [APOC plugin](https://github.com/neo4j-contrib/neo4j-apoc-procedures) also needs to be updated.

Reply:
I did not. We can implement a functional test.

The APOC plugin is picked up by neo4j automatically.  
https://neo4j.com/labs/apoc/4.0/installation/#docker

Reply:
@lioramilbaum Could you please confirm that is the case for 3.x too?

Reply:
```Status: Downloaded newer image for neo4j:3.5.26
Creating db ... done
Creating api ... done
(estuary) [lmilbaum@lmilbaum estuary-api]$ docker logs db
Fetching versions.json for Plugin 'apoc' from https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json
Installing Plugin 'apoc' from https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/3.5.0.15/apoc-3.5.0.15-all.jar to /var/lib/neo4j/plugins/apoc.jar 
Applying default values for plugin apoc to neo4j.conf
Skipping dbms.security.procedures.unrestricted for plugin apoc because it is already set
Active database: graph.db
Directories in use:
  home:         /var/lib/neo4j
  config:       /var/lib/neo4j/conf
  logs:         /logs
  plugins:      /var/lib/neo4j/plugins
  import:       /var/lib/neo4j/import
  data:         /var/lib/neo4j/data
  certificates: /var/lib/neo4j/certificates
  run:          /var/lib/neo4j/run
Starting Neo4j.
2021-03-23 06:46:22.392+0000 INFO  ======== Neo4j 3.5.26 ========
2021-03-23 06:46:22.404+0000 INFO  Starting...
2021-03-23 06:46:29.807+0000 INFO  Bolt enabled on 0.0.0.0:7687.
2021-03-23 06:46:31.117+0000 INFO  Started.
2021-03-23 06:46:32.029+0000 INFO  Remote interface available at http://localhost:7474/```

Reply:
Added another workflow - infra, for testing infra issues. Also added a test which validates the plugin is installed.